blog/2017-05-31-devstudio-survey.adoc

```diff
@@ -0,0 +1,18 @@
+= Are you a Developer User Experience lead ?
+:page-date: 2017-05-31
+:page-layout: blog
+:page-author: mhusnain
```

Comment:
Thank you, @jeffmaury. Added now. Could you merge this, if you have the rights? I think @xcoulon might be away on PTO or something.